kani-compiler/src/codegen_boogie/context/boogie_ctx.rs

```diff
@@ -324,47 +324,35 @@ impl<'tcx> BoogieCtx<'tcx> {
         }
     }
 
-    /// Write the program to the given writer
-    pub fn write<T: Write>(&self, writer: &mut T) -> std::io::Result<()> {
-        self.program.write_to(writer)?;
-        Ok(())
-    }
-
-    fn operand_ty(&self, local_decls: &LocalDecls<'tcx>, o: &Operand<'tcx>) -> Ty<'tcx> {
+    fn operand_ty(&self, o: &Operand<'tcx>) -> Ty<'tcx> {
         // TODO: monomorphize
```

Comment:
if you move to StableMIR you won't need to monomorphize. :smile:

Reply:
Nice! I need to educate myself on StableMIR!